static/js/typing_data.js

```diff
@@ -51,11 +51,17 @@ export function get_group_typists(group) {
     return typist_dct.get(key) || [];
 }
 
-export function get_all_typists() {
-    let typists = [].concat(...Array.from(typist_dct.values()));
-    typists = util.sorted_ids(typists);
-    typists = _.sortedUniq(typists);
-    return typists;
+export function get_all_pms_typists() {
+    let pms_typists = [];
+    for (const [key, typists] of typist_dct.entries()) {
+        if (!key.startsWith("{")) {
```

Comment:
Can you add a comment on why this check will produce correct results?  It strikes me as a bit hacky.

Reply:
It could've been written better as:
```
        try {
            JSON.parse(key);  // Key is a JSON string for streams typing
        }
        catch {
            pms_typists = pms_typists.concat(typists);
        }
```

Changed.